xyz-to-coords.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+def give_geometry(filename):
+    cwd = os.getcwd()
+    coords = []
+    with open(os.path.join(cwd, "geometries", filename)) as f:
+            try:
+                num_atoms = 0
+                counter = 0
+                for line in f:
+                    counter += 1
+                    if counter == 0:
+                        num_atoms = int(line.split()[0])
+                    elif counter == 2:
+                        continue
+                    elif counter > 2:
+                        element = line.split()
+                        symbol = element[0]
+                        x,y,z = float(element[1]), float(element[2]), float(element[3])
+                        coords.append([symbol, x, y, z])
+                    elif counter == 2+num_atoms:
+                        break
+            except:
+                counter = 0
+                natoms = 0
+                for line in f:
+                    if counter == 0:
+                        natoms = int(line.split()[0])
+                    elif counter == 4 + natoms:
+                        break
+                    elif counter > 0:
+                        atom = line.split()
+                        at_sym = atom[3]
+                        x,y,z = float(atom[0]), float(atom[1]), float(atom[2])
+                        coords.append([at_sym, x, y, z])
+                    counter += 1
+    return coords
```

Comment:
This works for my .xyz and my .sdf files. Notably I had to alter the counter on line 27 and 31 to be at zero.

Reply:
Why are you using try/except to jump between cases and not an if statement?

Reply:
I can change to an if statement. I don't have an inclination toward try/except. In terms of speed if is probably faster as compared to my code when an exception is raised for .sdf files.

Reply:
Changed

Reply:
I wouldn't worry about speed until it becomes an issue.